README.md

```diff
@@ -152,31 +150,37 @@ add_action('Flynt/afterRegisterComponents', function () {
                 'type' => 'flexible_content',
                 'button_label' => 'Add Component',
                 'layouts' => [
-                    Api::loadFields('BlockWysiwyg', 'layout'),
-                ],
-            ],
+                    Components\BlockWysiwyg\getACFLayout(),
+                ] 
+            ]
         ],
         'location' => [
             [
                 [
                     'param' => 'post_type',
                     'operator' => '==',
-                    'value' => 'page',
+                    'value' => 'page'
                 ],
-            ],
-        ],
+                [
+                    'param' => 'page_type',
+                    'operator' => '!=',
+                    'value' => 'posts_page'
```

Comment:
@emcarru What do you think about removing the whole location part, just to keep it short? Or replace it with [...] or something?

Reply:
I wanted to have one full example that you could in theory copy and paste and it would work. I could make it shorter by removing the second argument though (so just keep post_type == page). What do you think?

Reply:
Ok, get it. I thought that "real" examples are in the code already. But fine to keep it the way it is if you prefer it.